src/chrome/content/rules/jekyllrb.com.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+    Redirection:
+        www.jekyllrb.com -> jekyllrb.com
+-->
+<ruleset name="Jekyll">
+    <target host="jekyllrb.com" />
+    <target host="www.jekyllrb.com" />
+
+    <securecookie host="^(www\.)?jekyllrb\.com$" name=".+" />
+
+    <!-- Fix this redirection -->
+    <rule from="^http(s)?://jekyllrb.com/"
+            to="https://jekyllrb.com/" />
```

Comment:
> Perhaps you mean the https://www.jekyllrb.com -> http://jekyllrb.com issue?

Exactly.

Maybe I have to change line 12 to www. as that's of course the thing where it has to redirect.